example/android/build.gradle

```diff
@@ -12,10 +12,13 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath("com.android.tools.build:gradle:3.5.3")
+        classpath('com.android.tools.build:gradle:4.1.2')
 
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
+
+        classpath 'com.quinn.hunter:hunter-okhttp-plugin:1.1.0'
+        classpath 'com.quinn.hunter:hunter-transform:1.1.0'
```

Comment:
So the RN code says not to put dependencies here, but this fails when I place it in the individual module - any thoughts around this would be great.

Reply:
What error message do you get when it failed ?